src/main/java/eu/europeana/metis/sandbox/config/amqp/AmqpConfiguration.java

```diff
@@ -189,7 +189,7 @@ Declarables queues() {
                     .deadLetterRoutingKey(mediaProcessedDlq).build(),
         QueueBuilder.durable(publishedQueue).deadLetterExchange(exchangeDlq)
                     .deadLetterRoutingKey(publishedDlq).build(),
-        QueueBuilder.durable(deBiasReadyQueue).deadLetterExchange(deBiasReadyDlq)
+        QueueBuilder.durable(deBiasReadyQueue).deadLetterExchange(exchangeDlq)
```

Comment:
the queue name has to be different otherwise will mix messages from normal workflow with debias workflow.

Reply:
fixed bug indeed.